src/asm/rgbasm.5

```diff
@@ -812,6 +812,66 @@ first, followed by the one from
 and the one from
 .Ql bar.o
 last.
+.Ss Inline Fragments
+Inline fragments are useful for short blocks of code or data that are only referenced once.
+They are section fragments created by surrounding instructions or directives with
+.Ql [[
+double brackets
+.Ql ]] ,
+without a separate
+.Ic SECTION FRAGMENT
+declaration.
+.Pp
+The content of an inline fragment becomes a section fragment, sharing the same constraints as its parent ROM section.
```

Comment:
This doesn't make sense: if the parent is fixed at address `$1234`, the fragment can't be fixed at this address too.

Reply:
True, it shares the alignment and bank constraints, but not the fixed address.

Reply:
Why share the alignment constraint?

Reply:
...never mind, it does not share the alignment, only the bank. I was thinking in terms of older rgbasm where you would declare multiple `SECTION FRAGMENT "name", ROMX[$abcd], BANK[$02], ALIGN[4]` and they would just get concatenated and then constrained altogether.